api/ca.proto

```diff
@@ -22,8 +22,9 @@ service NodeCA {
 	rpc NodeCertificateStatus(NodeCertificateStatusRequest) returns (NodeCertificateStatusResponse) {
 		option (docker.protobuf.plugin.tls_authorization) = { insecure: true };
 	};
-	// GetEncryptionConfig returns the current encryption configuration for the cluster
-	rpc GetEncryptionConfig(GetEncryptionConfigRequest) returns (GetEncryptionConfigResponse) {
+	// GetUnlockKey returns the current unlock key for the cluster for the role of the client
+	// asking.
+	rpc GetUnlockKey(GetUnlockKeyRequest) returns (GetUnlockKeyResponse) {
```

Comment:
Since `GetUnlockKey` returns the unlock key from the cluster object, do you think it makes sense to redact it from the object returned by `GetCluster` and `ListClusters`? @diogomonica?